lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -41,3 +41,10 @@ export type ActiveAccountFilters = {
     homeAccountId: string;
     localAccountId: string;
 };
+
+export function updateAccountInfo(
+    accountInfo: AccountInfo,
+    updatedAttributes: Partial<AccountInfo>
+): AccountInfo {
+    return { ...accountInfo, ...updatedAttributes };
```

Comment:
This feels basic enough to not need a helper method

Reply:
Yeah originally I thought it'd be more complex, removing.